requirements/run.in

```diff
@@ -1,21 +1,21 @@
 # Since the install procedure install the packages below from pypi, we cannot
 # pin versions outside pypi, i.e. latest git master branch version.
-ipython==8.1.1
+ipython==8.22.1
 lockfile==0.12.2
 python-openflow
-python-daemon==2.3.1
+python-daemon==3.0.1
 janus==1.0.0
-jinja2==3.1.2
-watchdog==2.1.9
-pyjwt==2.4.0
-pymongo==4.1.0
-pydantic==1.9.0
-dnspython==2.2.1
-email-validator==1.3.0
-tenacity==8.0.1
-elastic-apm==6.9.1
-openapi-core==0.16.6
-httpx==0.24.0
-starlette[full]==0.26.0
-uvicorn[standard]==0.21.1
-asgiref==3.6.0
+jinja2==3.1.3
+watchdog==4.0.0
+pyjwt==2.8.0
+pymongo==4.6.2
+pydantic==2.6.3
+dnspython==2.6.1
+email-validator==2.1.1
+tenacity==8.2.3
+elastic-apm==6.20.0
+openapi-core==0.18.2
+httpx==0.27.0
+starlette[full]==0.31.1
```

Comment:
Could not update `starlette` to last version because of `openapi-core==0.18.2` needs.